languages/elixir/exercises/concept/errors/.docs/instructions.md

```diff
@@ -0,0 +1,51 @@
+While working at _Instruments of Texas_, you are tasked to work on an experimental calculator written in Elixir. Your team is having a problem with some operations raising errors and crashing the process. You have been tasked to write a function which wraps the operation function so that the errors can be handled more elegantly with idiomatic Elixir code.
+
+## 1. Warn the team
+
+Implement the function `calculate!/2` to call the operation function with the stack as the only argument. The operation function is defined else where, but you know that it can either complete successfully or raise an error.
```

Comment:
```suggestion
Implement the function `calculate!/2` to call the operation function with the stack as the only argument. The operation function is defined elsewhere, but you know that it can either complete successfully or raise an error.
```